packages/plugin/webpack/src/WebpackPlugin.ts

```diff
@@ -289,20 +293,18 @@ Your packaged app may be larger than expected if you dont ignore everything othe
 
       const config = await this.configGenerator.getRendererConfig(this.config.renderer.entryPoints);
       const compiler = webpack(config);
+      const app = express();
       const server = webpackDevMiddleware(compiler, {
-        logger: {
-          debug: tab.log.bind(tab),
-          log: tab.log.bind(tab),
-          info: tab.log.bind(tab),
-          error: tab.log.bind(tab),
-          warn: tab.log.bind(tab),
-        },
         publicPath: '/',
-        hot: true,
-        historyApiFallback: true,
         writeToDisk: true,
-      } as any);
-      const app = express();
+      });
+      server.context.logger = {
```

Comment:
this line have an error tips in my local development env: `TS2339: Property 'logger' does not exist on type 'Context'.`
how can I fix it? thx

Reply:
This is closed in favor of replacing express custom server with webpack-dev-server #2225

Reply:
I just turn webpack-dev-middleware's version 4 to 3,  param error has gone.
i guess the dep-bot auto update webpack-dev-middleware's version, occur the error in webpack 5

Reply:
I copy ElectronLogging to my local env。
It have a problem, the order of console log be chaos。

Reply:
> I just turn webpack-dev-middleware's version 4 to 3, param error has gone.
> i guess the dep-bot auto update webpack-dev-middleware's version, occur the error in webpack 5

finally, I fix my problems.
webpack-dev-middleware's version 4 to 3, will make webpack-dev-middleware cann't work 
use yours plugin have two problem.

1. the order of console log is chaos.
2. webpackDevServer beta version is weak

current my solutions：use spawn and webpack-cli can use webpack 5 and webpackDevServer 3 . its work well.